integrated_channels/integrated_channel/exporters/content_metadata.py

```diff
@@ -201,6 +203,8 @@ def _check_matched_content_updated_at(
                     remote_deleted_at__isnull=True,
                     remote_created_at__isnull=False,
                 )
+                content_query.add(Q(remote_errored_at__lt=self.LAST_24_HRS) | Q(
+                    remote_errored_at__isnull=True), Q.AND)
```

Comment:
shouldn't the query be `remote_errored_at__gt=self.LAST_24_HRS` or am I missing something here? 🤔

Reply:
Its basically excluding records which errored in last 24 hrs. That's why `errored_at < last 24hrs` 🤔 

Or other way to read this is `errored_at < one_day_ago`